src/Layout/redist/targets/GenerateLayout.targets

```diff
@@ -30,6 +30,18 @@
     <Delete Files="@(RoslynFrameworkAssemblies)" />
   </Target>
 
+  <Target Name="PublishNETAnalyzers"
+          BeforeTargets="Build;PublishNetSdk">
+     <PropertyGroup>
+      <AnalyzersDirectory>$(ArtifactsBinDir)$(Configuration)\Sdks\Microsoft.NET.Sdk\analyzers</AnalyzersDirectory>
+    </PropertyGroup>
+    <ItemGroup>
+      <AnalyzerAssemblies Include="$(NuGetPackageRoot)Microsoft.NetCore.Analyzers/$(MicrosoftNetCoreAnalyzersVersion)/analyzers/dotnet/**/*" />
```

Comment:
@dsplaisted It appears this is failing on ubuntu runs. I'll investigate but I would appreciate it if this is a failure that you've seen before. I assume it has to do with the location of the packages folder in linux vs windows?

Reply:
nevermind, I got it. its filesystem case-sensitivity